src/config/translations/fr-FR.js

```diff
@@ -32,5 +32,9 @@ export default {
         male: 'Homme',
         female: 'Femme',
         unknown: 'Inconnu'
+    },
```

Comment:
it's a good i18n key, you have to indicate the name of the form. It's specific to one form.